tests/common/ixia/ixia_fixtures.py

```diff
@@ -0,0 +1,141 @@
+"""This module contains the necessary fixtures for running test cases with
+ Ixia devices and IxNetwork. If more fixtures are required, they should be 
+included in this file.
+"""
+
+import pytest
+import pprint
+from common.devices import FanoutHost
+from ixnetwork_restpy import SessionAssistant, Files
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_ip(testbed):
+    """ 
+    In an Ixia testbed, there is no PTF docker. 
+    Hence, we use ptf_ip field to store Ixia API server. 
+    This fixture returns the IP address of the Ixia API server.
+
+    Args: 
+       testbed (pytest fixture): The testbed fixture.
+
+    Returns:
+        Ixia PTF server (API server) IP
```

Comment:
There is no need to mention PTF server here. Just say 'IP of Ixia API server'

Reply:
Removed the “PTF Server” from the comment.